### PR TITLE
add eth_signTypedData_v4

### PR DESCRIPTION
**Pull request by adonesky1**
Resolves: #94

### CODE REVIEW

File: openrpc.json [hunk 1]
```diff
@@ -64,9 +64,7 @@
               "value": {
                 "chainId": "0x64",
                 "chainName": "Gnosis",
-                "rpcUrls": [
-                  "https://dai.poa.network"
-                ],
+                "rpcUrls": ["https://dai.poa.network"],
```
Comment (adonesky1) [inline openrpc.json:67]: sorry for these random formatting changes... I can revert them if that's desired

**Review by shanejonas — APPROVED** [review]
Looks good

File: openrpc.json [hunk 2]
```diff
@@ -391,6 +383,95 @@
...
+                    "wallet": "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB"
+                  },
+                  "contents": "Hello, Bob!"
+                }
+              }
+            }
+          ],
+          "result": {
+            "name": "Signature",
+            "value": "[the signature]"
```
Comment (shanejonas) [inline openrpc.json:470]: could make this an example signed result

Comment (shanejonas) [inline openrpc.json:470]:
```suggestion
            "value": "0x4355c47d63924e8a72e509b65029052eb6c299d53a04e167c5775fd466751c9d07299936d304c153f6443dfa05f40ff007d72911b6f72307f996231605b915621c"
```

**Review by shanejonas — APPROVED** [review]

**Pull request merged**